front/src/modules/activities/tasks/hooks/useTasks.ts

```diff
@@ -110,10 +110,15 @@ export function useTasks(entity?: ActivityTargetableEntity) {
 
   const completedTasks = completeTasksData?.findManyActivities;
 
+  const dueTasks = todayOrPreviousTasks?.filter(
```

Comment:
The naming should be different because we expect to have an array of task and it's a number.

Maybe `dueTaskCount` ?

Reply:
Okay